table/options.go

```diff
@@ -376,6 +376,12 @@ func (m Model) WithFilterFunc(shouldInclude func(row Row, filterInput string) bo
 	return m
 }
 
+func (m Model) WithFuzzyFilter() Model {
```

Comment:
Would prefer a bool parameter to toggle on/off. Otherwise to remove the fuzzy filter you'd need to do `WithFilterFunc(nil)` which feels odd. Either this or we keep NewFuzzyFilter and have the user do:

```go
myTable.WithFilterFunc(NewFuzzyFilter())
```

Reply:
Let me know if the approach I chose is fine :)